pkgs/top-level/all-packages.nix

```
@@ -3377,6 +3377,8 @@ with pkgs;
 
   authelia = callPackage ../servers/authelia { };
 
+  authentik-outposts = recurseIntoAttrs (callPackage ../by-name/au/authentik/outposts.nix { });
```

```suggestion
  authentik-outposts = recurseIntoAttrs (callPackages ../by-name/au/authentik/outposts.nix { });
```
otherwise overrides don't work